CODEOWNERS

```diff
@@ -0,0 +1 @@
+/PULL_REQUEST_TEMPLATE.md @opendoor-labs/cloud-infra
```

Comment:
Is anyone able to add this repo to https://github.com/orgs/opendoor-labs/teams/cloud-infra/repositories ? Looks like I lack permission.

Reply:
I just did. Thx!

Reply:
thanks Xinxing! 🙌